### ISSUE TITLE
HtmlNode.CreateNode add closing tag to video source item

### ISSUE THREAD

**Issue by valse** [zzzprojects/html-agility-pack#63]
Hi,
   if I try to create a node from this html string:
```
<video width="1280" height="720" poster="poster.jpg">
   <source type="video/mp4" src="video.mp4">
</video>
```
the result outer html is like this:
```
<video width="1280" height="720" poster="poster.jpg">
   <source type="video/mp4" src="video.mp4"></source>
</video>
```
that's not valid.

Thanks
Marco

**Comment by JonathanMagnan** [comment]
Hello @valse ,

Thank you for reporting,

We will look at this issue very soon.

Best Regards,

Jonathan

**Comment by JonathanMagnan** [comment]
Hello @valse ,

We fixed the source tag in the v1.5.2-beta5,

We will soon re-work the library to fix all tags with a similar issue.

Feel free to re-open if you still have the issue.

Best Regards,

Jonathan
